src/Entity/FileLoader.php

```diff
@@ -62,6 +75,73 @@ public function fromEntity(FilePropertyConfiguration $configuration, object $ent
             return null;
         }
 
-        return $this->fileManager->load($configuration->getFileSystemName(), $path);
+        $file = $this->fileManager->load($configuration->getFileSystemName(), $path);
+        if (null !== $file) {
+            $this->checkFileExistenceIfEnabled($className, $file);
+        }
+
+        return $file;
+    }
+
+    public function disableFileExistanceChecksOnLoad(): void
+    {
+        $this->disableFileExistenceChecksOnLoad();
+    }
+
+    public function enableFileExistanceChecksOnLoad(): void
```

Comment:
this method with misspelled name can be removed

Reply:
These checks are turned off only during loading files, so without it it would be confusing.